backend/src/main/kotlin/org/loculus/backend/model/ReleasedDataModel.kt

```diff
@@ -93,6 +98,14 @@ open class ReleasedDataModel(
             NullNode.getInstance()
         }
 
+        backendConfig.
+
+        // TODO in here we can calculate the earlierstReleaseDate and add it to the metadata
+        // Settings are pulled from backendConfig
+        // Or maybe we can't do it here, because we can't do it on a line-by-line basis?
+        // is it  sufficient to just take the earliest of 'releasedAtTimestamp' and any other
```

Comment:
There's this other element that it's across versions – for version 3 it will be version 1s release date, or the earliest of the external fields, whichever is earliest